app/src/main/java/com/microsoft/device/samples/dualscreenexperience/presentation/catalog/ui/view/FirstPageContent.kt

```diff
@@ -0,0 +1,218 @@
+import com.microsoft.device.samples.dualscreenexperience.presentation.catalog.utils.contentDescription
+import com.microsoft.device.samples.dualscreenexperience.presentation.util.sizeOrZero
+
+const val FIRST_PAGE_FIRST_TEXT_ID = "firstText"
+const val FIRST_PAGE_SECOND_TEXT_ID = "secondText"
+const val FIRST_PAGE_THIRD_TEXT_ID = "thirdText"
+const val FIRST_PAGE_FOURTH_TEXT_ID = "fourthText"
+const val FIRST_PAGE_FIFTH_TEXT_ID = "fifthText"
+
+@Composable
+fun CatalogFirstPage(
+    modifier: Modifier = Modifier,
+    catalogList: List<CatalogItem>,
+    onItemClick: (Int) -> Unit
+) {
+    val pageNumberOrdinal = CatalogPage.Page1.ordinal
+    val catalogItem = catalogList[pageNumberOrdinal]
+
+    PageLayout(
+        modifier,
+        pageNumberOrdinal + 1,
+        catalogList.sizeOrZero()
+    ) {
+        TableOfContents(
+            modifier = Modifier.fillMaxHeight(),
+            catalogItem = catalogItem,
+            onItemClick = onItemClick
+        )
+    }
+}
+
+@Composable
+private fun getConstraintSetForTableOfContents() = ConstraintSet {
+    val firstTextRef = createRefFor(FIRST_PAGE_FIRST_TEXT_ID)
+    val secondTextRef = createRefFor(FIRST_PAGE_SECOND_TEXT_ID)
+    val thirdTextRef = createRefFor(FIRST_PAGE_THIRD_TEXT_ID)
+    val fourthTextRef = createRefFor(FIRST_PAGE_FOURTH_TEXT_ID)
+    val fifthTextRef = createRefFor(FIRST_PAGE_FIFTH_TEXT_ID)
+
+    constrain(firstTextRef) {
+        start.linkTo(parent.start)
+        top.linkTo(parent.top)
+    }
+
+    constrain(secondTextRef) {
+        end.linkTo(parent.end)
+        start.linkTo(parent.start)
+        top.linkTo(firstTextRef.bottom)
+    }
+
+    constrain(thirdTextRef) {
+        end.linkTo(parent.end)
+        start.linkTo(parent.start)
+        top.linkTo(secondTextRef.bottom)
+    }
+
+    constrain(fourthTextRef) {
+        end.linkTo(parent.end)
+        start.linkTo(parent.start)
+        top.linkTo(thirdTextRef.bottom)
+    }
+
+    constrain(fifthTextRef) {
+        end.linkTo(parent.end)
+        start.linkTo(parent.start)
+        top.linkTo(fourthTextRef.bottom)
+    }
+}
+
+@Composable
+fun TableOfContents(
+    modifier: Modifier = Modifier,
+    catalogItem: CatalogItem,
+    onItemClick: (Int) -> Unit
+) {
+    val constraintSet = getConstraintSetForTableOfContents()
+
+    ConstraintLayout(
+        constraintSet = constraintSet,
+        modifier = modifier.fillMaxSize()
+    ) {
+        Text(
+            modifier = Modifier
+                .padding(
+                    start = dimensionResource(id = R.dimen.catalog_horizontal_margin),
+                    top = dimensionResource(id = R.dimen.catalog_margin_very_large),
+                    end = dimensionResource(id = R.dimen.catalog_horizontal_margin)
+                )
+                .layoutId(FIRST_PAGE_FIRST_TEXT_ID),
+            text = catalogItem.primaryDescription,
+            style = MaterialTheme.typography.h6.copy(color = MaterialTheme.colors.onSurface)
```

Comment:
I know we discussed this as a solution to not being able to include color in the Theme.kt file, but I realized you could also just pass the color (or text alignment or any other text style properties) straight to the Text composable:
```suggestion
            color = MaterialTheme.colors.onSurface,
            style = MaterialTheme.typography.h6
```